flow/endpoint.yaml

```diff
@@ -70,6 +70,15 @@ components:
           enum: [mesh, one_to_one]
           default: mesh
         tx_names:
+          description: |-
+            The system will use information from the supplied tx_names to populate the following
+            flow.packet.header fields if they are present:
+              - ethernet.dst using device ipv4 addresses from either interfaces, route ranges etc.
```

Comment:
shouldn't this description be in rx_names  and modified a little to imply "ethernet.dst using resolved mac addresses obtained using device ipv4 addresses from either interfaces, route ranges etc." ?